### PR TITLE
Improve docs, clean code after the review

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu [hunk 1]
```diff
@@ -29,10 +29,14 @@ Output of the decoder is on the GPU and uses `HWC` ordering.)code")
...
   .AddOptionalArg("device_memory_padding",
-      R"code(Padding for nvJPEG's device memory allocations.)code",
+      R"code(Padding for nvJPEG's device memory allocations.
+This parameter helps avoid reallocation in the nvJpeg whenever a bigger image
+is encountered and internal buffer needs to be reallocated to decode it. Default is 16MB.)code",
       16*1024*1024)
   .AddOptionalArg("host_memory_padding",
-      R"code(Padding for nvJPEG's host memory allocations.)code",
+      R"code(Padding for nvJPEG's host memory allocations.
+This parameter helps avoid reallocation in the nvJpeg whenever a bigger image
```
Comment (cliffwoolley) [inline dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: "reallocation in nvJPEG" for both lines 33 and 38.

Comment (JanuszL) [reply dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: Done

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu [hunk 2]
```diff
@@ -29,10 +29,14 @@ Output of the decoder is on the GPU and uses `HWC` ordering.)code")
   .AddOptionalArg("use_batched_decode",
       R"code(Use nvJPEG's batched decoding API.)code", false)
   .AddOptionalArg("device_memory_padding",
-      R"code(Padding for nvJPEG's device memory allocations.)code",
+      R"code(Padding for nvJPEG's device memory allocations.
+This parameter helps avoid reallocation in nvJPEG whenever a bigger image
```
Comment (klecki) [inline dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: This parameter helps _to_ avoid reallocations in ...

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu [hunk 3]
```diff
@@ -29,10 +29,14 @@ Output of the decoder is on the GPU and uses `HWC` ordering.)code")
...
+is encountered and internal buffer needs to be reallocated to decode it. Default is 16MB.)code",
       16*1024*1024)
   .AddOptionalArg("host_memory_padding",
-      R"code(Padding for nvJPEG's host memory allocations.)code",
+      R"code(Padding for nvJPEG's host memory allocations.
+This parameter helps avoid reallocation in nvJPEG whenever a bigger image
```
Comment (klecki) [inline dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: This parameter helps _to_ avoid reallocations in ...

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu (hunk 2 above)
Comment (JanuszL) [reply dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: Done

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu (hunk 3 above)
Comment (JanuszL) [reply dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: Done

**Pull request merged**